node/node.go

```diff
@@ -1030,6 +1030,9 @@ func New(
 			txPoolConfig.AccountSlots = harmonyconfig.TxPool.AccountSlots
 			txPoolConfig.GlobalSlots = harmonyconfig.TxPool.GlobalSlots
 			txPoolConfig.Locals = append(txPoolConfig.Locals, localAccounts...)
+			txPoolConfig.AccountQueue = harmonyconfig.TxPool.AccountQueue
+			txPoolConfig.GlobalQueue = harmonyconfig.TxPool.GlobalQueue
+			txPoolConfig.Lifetime = harmonyconfig.TxPool.Lifetime
```

Comment:
Is `PriceLimit` and `PriceBump` not assigned in `txPoolConfig` here? A couple of lines above it is hardcoded as well.

Reply:
fixed